CHANGES.md

```diff
@@ -1,6 +1,8 @@
 Unreleased
 ----------
 
+- Remove progress bar before executing the command for `dune exec` (#5370, @bobot)
```

Comment:
Are you sure this bug is present in 2.9? I recall we messed around with the progress bar quite a bit for 3.0, so perhaps we introduced it since?